### PR TITLE
Fix share icons

### PR DESCRIPTION
**Pull request by henrytao-me**
This PR fixes Android share icons as well as Toolbar title. 

Resolves https://github.com/CovidShield/mobile/issues/63

## Note:
- In before image, the background push toolbar to the top which is wrong. 
- In after image, the content is overlapped with the background. It depends on device height. The content and share options are always stick to the bottom.
- ScrollView is removed from this view because it isn't necessary.

### CODE REVIEW

File: src/screens/sharing/Sharing.tsx [hunk 1]
```diff
@@ -53,23 +59,23 @@ export const SharingScreen = () => {
           navLabel={i18n.translate('Sharing.Close')}
           onIconClicked={close}
         />
-        <ScrollView>
```
Comment (jordanAtShopify) [inline src/screens/sharing/Sharing.tsx:56]: Think we need to keep ScrollView here too for accessibility

Comment (henrytao-me) [reply src/screens/sharing/Sharing.tsx:56]: Added it back!

**Comment by henrytao-me** [comment]
Ready for reviewing 🙇

**Comment by kpeatt** [comment]
@henrytao-me I think the text should always be below the image. It's okay if this page scrolls on smaller devices.

**Comment by henrytao-me** [comment]
It's difficult to balance both. In this case, if we want to text is always below image, then this screen can scroll just a little bit which is odd. 

One option is that we can make the svg connect-able on both sides (left and right), so that we can 
scale the height to whatever space lefts and fill up the width with connected left and right. What do you think?

image

**Comment by AdamWhitcroft** [comment]
@henrytao-me I think let's keep things a little simpler and just let the view scroll as much as it needs to - keeping the illustration as is. Is that OK?

**Comment by kpeatt** [comment]
Share icons work great in Android simulator

**Comment by kpeatt** [comment]
Works great on Pixel 4 device as well.

**Comment by henrytao-me** [comment]
Pushed the code to render top down. It looks like this at least on Pixel One.

**Comment by henrytao-me** [comment]
Merge now. I will do a follow PR if needed.

**Pull request merged**